theano/sandbox/cuda/__init__.py

```diff
@@ -362,6 +362,9 @@ def c_libraries(self):
     def c_lib_dirs(self):
         return [config.dnn.library_path]
 
+    def c_compile_args(self):
+        return ['-rpath', config.dnn.library_path]
```

Comment:
It is not sure the order is kept. Why not use ['-rpath '+config...] instead? Just for safety that we don't change the order of arguments